docs/applications/development/clojure-deployment-with-immutant-and-wildfly-on-ubuntu-14-04.md

```diff
@@ -231,9 +231,9 @@ To deploy Clojure application with WildFly you will need to install Immutant lei
 	}
 	~~~
 
-	This sets the destination folder to copy the WAR file into, context path and WAR file name - which should be ROOT for root context path.
+	This sets the destination folder into which the WAR file, context path and WAR file name are copied - which should be ROOT for root context path.
```

Comment:
Yea, that was a flub on my part. Do you want to merge this PR, and then make the changes discussed in these notes? Once you do, add an update on your PR to Linode/Docs and we'll proceed.